opsmngr/automation_config.go

```diff
@@ -411,6 +411,7 @@ type Process struct {
 	BackupRestoreFilterList                    interface{}        `json:"backupRestoreFilterList,omitempty"`
 	BackupRestoreJobID                         string             `json:"backupRestoreJobId,omitempty"`
 	BackupRestoreURL                           string             `json:"backupRestoreUrl,omitempty"`
+	BackupRestoreIsSuccessiveUpgrade           string             `json:"backupRestoreIsSuccessiveUpgrade,omitempty"`
```

Comment:
isn't this a boolean field in mms?

Reply:
Yes, sorry about that and thanks for spotting that! eeec947b82592f6f0a9a6f5c89c6cec33993f5d1